### PR TITLE
XRDDEV-797: Implement UI for viewing endpoints of a service

### PR DESCRIPTION
**Pull request by TJaakkola**

### CODE REVIEW

File: src/proxy-ui-api/frontend/src/views/Service/Parameters/ServiceParameters.vue [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+    return {
+      touched: false,
+      confirmGroup: false,
+      confirmMember: false,
+      confirmAllSubjects: false,
+      selectedMember: undefined as NullableSubject,
+      description: undefined,
+      url: '',
+      addSubjectsDialogVisible: false,
+      timeout: 23,
```
Comment (Riippi) [inline src/proxy-ui-api/frontend/src/views/Service/Parameters/ServiceParameters.vue]: Timeout is hardcoded  to 23 ?

File: src/proxy-ui-api/frontend/src/views/Service/Parameters/ServiceParameters.vue [hunk 2]
```diff
@@ -0,0 +1,488 @@
...
+  },
+  watch: {
+    isHttps(val) {
+      // If user edits http to https --> change "ssl auth" to true
+      if (val === true) {
+        this.service.ssl_auth = true;
+      }
+    },
+  },
+  created() {
```
Comment (Riippi) [inline src/proxy-ui-api/frontend/src/views/Service/Parameters/ServiceParameters.vue]: If this created is not used it can be deleted.

Comment (TJaakkola) [reply src/proxy-ui-api/frontend/src/views/Service/Parameters/ServiceParameters.vue]: This haven't been changed. Don't actually know what is this timeout variable for but it hasn't changed in this task (just moved to another file)

File: src/proxy-ui-api/frontend/src/views/Service/Parameters/ServiceParameters.vue (hunk 1 above)
Comment (TJaakkola) [reply src/proxy-ui-api/frontend/src/views/Service/Parameters/ServiceParameters.vue]: This haven't been changed. Don't actually know what is this timeout variable for but it hasn't changed in this task (just moved to another file)

**Pull request merged**